### DISCUSSION TITLE
v2.6.0

### DISCUSSION THREAD

**Discussion by KodeStar** [linuxserver/Heimdall#1292] [Announcements]
This release updated the underlying framework from Laravel 8 to Laravel 10.

In settings there is a new option called Treat Tags As with the options Folders, Tags and Categories.

**Folders**: This is the same as the current functionality.
**Tags**: This displays all the pinned items on the dashboard and has a list of tags at the top, clicking on a tag will only show items with that tag.
**Categories**: This displays pinned items in columns grouped by the tag.

<img width="1834" alt="Screenshot 2024-02-18 at 21 01 21" src="https://github.com/linuxserver/Heimdall/assets/5698566/a6f5f8f6-5b30-4a90-91d1-fdae6b6f8069">
<img width="1834" alt="Screenshot 2024-02-18 at 21 01 11" src="https://github.com/linuxserver/Heimdall/assets/5698566/4af43a78-5511-4093-a6d8-3148696675aa">
<img width="1834" alt="Screenshot 2024-02-18 at 21 02 02" src="https://github.com/linuxserver/Heimdall/assets/5698566/a144011e-ffd9-45fc-b1cd-dbd528b022a0">

**Comment by Drexlin** [comment]
I really like the 3rd option ("Categories" option), but is there a way to also have it display anything with the "Home dashboard" tag in a separate list as well? Thanks!

**Comment by kolik2976** [comment]
Hello, is it possible to make the website icon only visible, without a name? Something like that https://dl.dropboxusercontent.com/scl/fi/wcetmiigk1he0nl21vfw4/catsxp_IiFG1A7yaS.png?rlkey=28jervdi8skbf37u27g4f9yq3&dl=0
